intake_stac/tests/test_catalog.py

```diff
@@ -25,6 +32,16 @@ def stac_collection_obj():
     )
 
 
+@pytest.fixture(scope="module")
+def stac_item_collection_obj():
+    # TODO: change load -> open,
+    # see https://github.com/sat-utils/sat-stac/issues/52
+    # use github/sat-utils/sat-stac/master/test/items.json
+    return satstac.ItemCollection.load(
+        os.path.join(os.path.dirname(__file__), "items.json")  # noqa: E501
+    )
```

Comment:
temporary work around here. See https://github.com/sat-utils/sat-stac/issues/52 for more detail.

Reply:
I've added this and release 0.3.1:
https://github.com/sat-utils/sat-stac/blob/master/CHANGELOG.md#v031---2019-12-06